hooks/useServer.ts

```diff
@@ -18,21 +18,294 @@ const MVP_ALLOWED_CHANNELS = {
   "govbot-testing": "959216625180098590"
 };
 
+const channels_response = {
```

Comment:
bump these two arrays of example data out to a fixtures.ts  file

Reply:
just saw it now. Probably can get rid of them entirely